terraform/backend.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+    backend "s3" {
+    bucket = "wordpress-s3"
```

Comment:
Could we use the parameters as variables?

Reply:
Yes definitely for the bucket name